--- a/numpy/_core/meson.build
+++ b/numpy/_core/meson.build
@@ -708,6 +708,7 @@
 
 # Build multiarray_tests module
 # -----------------------------
+if false
 py.extension_module('_multiarray_tests',
   [
     src_file.process('src/multiarray/_multiarray_tests.c.src'),
@@ -725,6 +726,7 @@
   subdir: 'numpy/_core',
   install_tag: 'tests'
 )
+endif
 
 _umath_tests_mtargets = mod_features.multi_targets(
   '_umath_tests.dispatch.h',
@@ -751,6 +753,7 @@
   ['_struct_ufunc_tests', 'src/umath/_struct_ufunc_tests.c', []],
   ['_operand_flag_tests', 'src/umath/_operand_flag_tests.c', []],
 ]
+if false
 foreach gen: test_modules_src
   py.extension_module(gen[0],
     gen[1],
@@ -763,6 +766,7 @@
     install_tag: 'tests'
   )
 endforeach
+endif
 
 # Build multiarray dispatch-able sources
 # --------------------------------------
